sources/api/openapi.yaml

```diff
@@ -324,3 +324,77 @@ paths:
           description: "Reboot requested"
         500:
           description: "Server error"
+
+  /actions/refresh-updates:
```

Comment:
Doesn't this still need a 423 lock error?  I don't think we'd want parallel attempts to refresh updates.